base/libc.jl

```diff
@@ -159,7 +159,7 @@ function strftime(fmt::AbstractString, tm::TmStruct)
     if n == 0
         return ""
     end
-    return String(pointer(timestr), n)
+    return unsafe_string(pointer(timestr), n)
```

Comment:
Looks like the same pattern is used in several other places. Should they also be changed?